be1-go/cli/organizer/organizer.go

```diff
@@ -20,7 +20,7 @@ var upgrader = websocket.Upgrader{
 }
 
 // Serve parses the CLI arguments and spawns a hub and a websocket server.
-func Serve(context *cli.Context) error {
+func OrganizerServe(context *cli.Context) error {
```

Comment:
I believe we can do without the name change since witness and organizer are in separate packages

Reply:
Yes good idea, I've changed that! Indeed that is the intention.